src/Components/Scenes/SideNavigation/FindDataListItem/FindDataListItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ListItem, ListItemText } from '@material-ui/core';
+import { Search } from '@material-ui/icons';
+import { Link, useLocation } from 'react-router-dom';
+
+const FindDataListItem = props => {
+  const { handleSearchClick, isSearchClicked } = props;
+  const activeLink = useLocation();
+  console.log('Clicked', isSearchClicked);
+
+  return (
+    <ListItem
+      component={Link}
+      to="/search"
+      onClick={() => handleSearchClick()}
+      button
+      className={
+        isSearchClicked === true || '/search' === activeLink.pathname
```

Comment:
If you want, can just do `isSearchClicked` without the comparison to `true`. Looks good though